File: Programming Basics/02-Conditional Statements - Exercise/shopping.js
function shopping(input) {

    const GPU = 250.00;

    let budget = Number(input[0]);
    let numberGpu = Number(input[1]);
    let numberCpu = Number(input[2]);
    let numberRam = Number(input[3]);

    let priceGpu = GPU * numberGpu;

    let cpu = priceGpu * 0.35;
    let ram = priceGpu * 0.10;

    let neededMoney = cpu * numberCpu + ram * numberRam + priceGpu;

    if(numberGpu > numberCpu) {
        neededMoney *= 0.85;
    }

    let total =Math.abs(budget - neededMoney);

    if(budget >= neededMoney){
        console.log(`You have ${total.toFixed(2)} leva left!`);
    } else {
        console.log(`Not enough money! You need ${total.toFixed(2)} leva more!`);
    }
}

shopping(["900",
"2",
"1",
"3"]);

File: Programming Basics Java/src/ConditionalStatementsExercise/Shopping.java
package ConditionalStatementsExercise;

import java.util.Scanner;

public class Shopping {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        int numberOfGPU = Integer.parseInt(scanner.nextLine());
        int numberOfCPU = Integer.parseInt(scanner.nextLine());
        int numberOfRAM = Integer.parseInt(scanner.nextLine());

        double priceForGPU = numberOfGPU * 250.00;
        double priceForCPU = numberOfCPU * (priceForGPU * 0.35);
        double priceForRAM = numberOfRAM * (priceForGPU * 0.10);

        double totalPrice = priceForGPU + priceForCPU + priceForRAM;

        if (numberOfGPU > numberOfCPU) {
            totalPrice *= 0.85;
        }

        double difference = Math.abs(budget - totalPrice);

        if (budget >= totalPrice) {
            System.out.printf("You have %.2f leva left!", difference);
        } else {
            System.out.printf("Not enough money! You need %.2f leva more!", difference);
        }
    }
}
